src/mote/modules/call.py

```diff
@@ -44,12 +43,9 @@ def fetch_channel_dict():
 def fetch_datetxt_dict(channel: str):
     try:
         datetxt_dict = {}
-        datelist = os.listdir("/srv/web/meetbot/%s" % channel)
+        datelist = f"{os.listdir(app.config['MEETING_DIR'])}/{channel}"
```

Comment:
This is not correct. `datelist` is supposed to be a list, now it's a string representing a list.
The next for-loop will not produce the expected result.
```suggestion
        datelist = os.listdir(f"{app.config['MEETING_DIR']}/{channel}")
```
You have the same mistake at different places in this file.

Reply:
@darknao ok I will resolve this